src/components/app/applicationHeader.tsx

```diff
@@ -40,6 +40,13 @@ function ActionCard() {
           </button>
         )}
       </div>
+      <div>
```

Comment:
Vi bør kanskje finne en ny plass for denne. Nå er toppmenyen høyere pga disse tre knappene (fikser senere)